great-idea-website/css/index.css

```diff
@@ -65,66 +65,107 @@ h1, h2, h3, h4, h5 {
 
 /* Your code starts here! */
 
-main {
+.main-content-area {
 	width: 900px;
 	display: block;
 	margin: auto;
+	margin-top: 20px;
 }
-header {
-	height: 80px;
-	padding: 5px;
+
+/* Main Navigation Styling */
+
+.main-nav {
 	margin: auto;
-	vertical-align: middle;
+	/*text-align: center;*/
+	width: 900px;
+	height: 100px;
+	position: relative;
 }
-nav {
+
+nav div {
 	display: inline-block;
+	/*height: 60px;*/
+	margin: 25px 50px 25px 0px;
 }
+
 nav a {
+	display: inline-block;
+	padding: 30px;
 	text-decoration: none;
 	color: gray;
 	font-size: 15px;
-	font-family: Helvetica;
-	width: 60px;
-	height: 20px;
-	padding: 20px;
+
+}
+
+nav a:first-child {
```

Comment:
nice job trying out a pseudo selector, but I don't see why you would ned this.  Just put a padding inside the entire nav element and you should be good to go.